src/stories/Goal/GoalItem.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { GoalItem } from '../../components/GoalItem';
+import { Tag } from '../../../graphql/@generated/genql';
+import { randomHex } from '../../utils/randomHex';
+import { accentIconColor } from '../../design/@generated/themes';
+import { backgroundColor as darkBackgroundColor } from '../../design/@generated/themes/dark.constants';
+import { backgroundColor as lightBackgroundColor } from '../../design/@generated/themes/light.constants';
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+    title: 'Taskany/GoalItem',
+    component: GoalItem,
+    args: {
+        title: 'Test goal',
+        id: 'TST-1',
+        stateHue: randomHex([darkBackgroundColor, lightBackgroundColor]),
```

Comment:
`hue` is the field of State model. Better to pass state as prop. Like `<GoalItem state={{ title, hue }}/>`

Reply:
done